index.js

```diff
@@ -48,12 +49,42 @@ module.exports = {
       ssb.metafeeds.findOrCreate(details, cb)
     }
 
-    function findOrCreateGroupFeed(input, cb) {
-      const secret = Buffer.isBuffer(input) ? new SecretKey(input) : input
+    function findEmptyGroupFeed(rootId, cb) {
+      console.log('in empty function')
+      pull(
+        ssb.metafeeds.branchStream({ root: rootId, live: false }),
+        pull.filter((branch) => branch.length === 4),
+        pull.map((branch) => branch[3]),
+        pull.filter((feed) => feed.recps),
```

Comment:
This filter could look into `feed.purpose` and test whether it matches something that looks like a group secret.

Reply:
checking that the length is 44 now